submissions/whatsapp-bot-node/readme.md

```diff
@@ -0,0 +1,52 @@
+# Chimoney Whatsapp Bot
+
+
+The Environment variables (.env) file contains:
+
+- `PORT`: Port on which the bot's HTTP server will run.
+- `BOT_URL`: Your whatsapp bot's URL.
+- `API_KEY`: Your API key for Chimoney integration.
+- `REDIRECT_URL`: Redirect URL for Chimoney transactions.
+- `CHIMONEY_BASE_URL`: Base URL for Chimoney operations.
+- `CHIMONEY_BASE_HOST`: Chimoney base host URL.
+- `DEFAULT_EMAIL`: Default email to receive transactions updates (since telegram doesn't have emails).
```

Comment:
What is this used for? You mention Telegram

Reply:
Is this a Whatsapp or Telegram bot?

Reply:
I made a mistake, it's whatsapp